code/lightna/lightna-webpack/App/Compiler/Webpack.php

```diff
@@ -200,13 +200,19 @@ protected function getModuleAlias(string $folder): string
     protected function saveConfig(): void
     {
         $this->build->putFile('webpack/webpack.config.js', $this->getWpConfigJs());
+        $this->build->putFile('webpack/webpack.config.dev.js', $this->getWpDevConfigJs());
     }
 
     protected function getWpConfigJs(): string
     {
         return 'module.exports = ' . json_pretty($this->getWpConfig());
     }
 
+    protected function getWpDevConfigJs(): string
+    {
+        return str_replace('building', 'build', $this->getWpConfigJs());
```

Comment:
@vesan-83 I guess some vars must be used here, right? 🙂

Reply:
we can use argv insude js to avoid change like that in php